src/fundus/parser/base_parser.py

```diff
@@ -172,7 +172,7 @@ def _search_members(cls, obj_type: type) -> List[Tuple[str, Any]]:
     @classmethod
     def attributes(cls) -> AttributeCollection:
         attrs: List[Attribute] = [
-            func for _, func in cls._search_members(Attribute) if func.__name__ not in ["__ld", "__meta"]
+            func for _, func in cls._search_members(Attribute) if func.__name__ not in ["__ld", "__meta", "free_access"]
```

Comment:
`free_access` should not be excluded here because it is a required attribute from the `attribute_guidelines`. If you check your branch's [`supported_publishers.md`](https://github.com/flairNLP/fundus/blob/add-free-access-atrribute/docs/supported_publishers.md) you will see that is flagged as missing for every publisher.

Reply:
I had to add this here, because otherwise the tests for the BaseParser would fail due to it expecting only 2 attributes. Should I instead modify the tests?

Reply:
Hmm, I'm undecided about what I want this to be like. On the one hand `BaseParser` was never supposed to support attributes also present in the guidelines natively and adding them somehow dilutes the design. On the other hand, manually adding free access to every parser is redundant and would serve no purpose.

The original design Idea was to have standardized attributes described in the `attribute_guidelines` which every parser had to add individually so that one could differentiate between the parser not supporting an attribute like `topics` and the parser no longer being able to parse said attribute.

`free_access` fits none of the above cases:
1. Being boolean, there is no information gained to know if it is supported by a specific parser or not. If it's not, simply yielding `True` already provides every piece of information.
2. It's not as standardized as the `meta` tag or `ld+json` section as we have seen with `Bild` so there is a need to support individual implementations.

I guess adding it to the guideline. treating it as every other attribute there while having a default in the `BaseParser`, and altering the tests would be a solution here.

Reply:
Ok, sounds good to me. I have updated the code.